app/views/katello/api/v2/products/index.json.rabl

```diff
@@ -3,5 +3,20 @@ object false
 extends "katello/api/v2/common/metadata"
 
 child @collection[:results] => :results do
-  extends "katello/api/v2/products/base"
+  collection @collection[:results]
+  attributes :id, :cp_id, :name, :label, :description
+
+  extends 'katello/api/v2/common/syncable'
+  extends 'katello/api/v2/common/org_reference'
+
+  attributes :provider_id
+  attributes :sync_summary
+
+  child :sync_plan do
+    attributes :name, :interval
+  end
+
+  node :repository_count do |product|
+    product.library_repositories.count
+  end
 end
```

Comment:
Ah okay. Not intending to go against any particular convention. Shoulda looked at the ``show`` template also before refactoring.

Reworking.